accname/name/comp_name_from_content.html

```diff
@@ -274,8 +274,19 @@ <h3 data-expectedlabel="one two three" data-testname="heading name from content
 <a href="#" data-expectedlabel="one two three" data-testname="link name from content for each child (no space, display:inline-block)" class="ex iblock"><span>one</span><span>two</span><span>three</span></a><br>
 <br>
 
+
+<!-- TODO: Add clarification to the Name from Content section of accname. -->
+<!--       If an element’s text node is transformed with CSS text-transform, -->
+<!--       what should be exposed to the a11y tree? The original text or the -->
+<!--       transformed text? These tests assume the *original* text. -->
+
+<h1 data-expectedlabel="Call us" data-testname="heading name from content with text-transform:none" class="ex" style="text-transform:none;">Call us</h1>
+<h1 data-expectedlabel="Call us" data-testname="heading name from content with text-transform:uppercase" class="ex" style="text-transform:uppercase;">Call us</h1>
+<h1 data-expectedlabel="Call us" data-testname="heading name from content with text-transform:capitalize" class="ex" style="text-transform:capitalize;">Call us</h1>
+<h1 data-expectedlabel="Call us" data-testname="heading name from content with text-transform:lowercase" class="ex" style="text-transform:lowercase;">Call us</h1>
```

Comment:
Follw-up from ARIA WG discussion. Match current (intentional) browser behavior.

```suggestion
<h1 data-expectedlabel="CALL US" data-testname="heading name from content with text-transform:uppercase" class="ex" style="text-transform:uppercase;">Call us</h1>
<h1 data-expectedlabel="Call Us" data-testname="heading name from content with text-transform:capitalize" class="ex" style="text-transform:capitalize;">Call us</h1>
<h1 data-expectedlabel="call us" data-testname="heading name from content with text-transform:lowercase" class="ex" style="text-transform:lowercase;">Call us</h1>
```